docs/source/tutorials/one_class_segmentation_2d_unet.rst

```diff
@@ -0,0 +1,223 @@
+One-class segmentation with 2D U-Net
+====================================
+
+In this tutorial we will learn the following features:
+
+- Training of a segmentation model (U-Net 2D) with a single label on multiple contrasts,
+
+- Testing of a trained model and computation of 3D evaluation metrics.
+
+- Visualization of the outputs of a trained model.
+
+
+Download dataset
+----------------
+
+We will use a publicly-available dataset consisting of MRI data of the spinal cord. This dataset is a subset of the
+`spine-generic multi-center dataset <https://github.com/spine-generic/data-multi-subject>`_ and has been pre-processed
+to facilitate training/testing of a new model. Namely, for each subject, all six contrasts were co-registered together,
+and semi-manual cord segmentation label was created. More details
+`here <https://github.com/ivadomed/ivadomed/blob/master/dev/prepare_data/README.md>`_.
+
+To download the dataset (~450MB), run the following commands in your terminal:
+
+.. code-block:: bash
+
+   # Download data
+   curl -o ivadomed_spinegeneric_registered.zip -L https://github.com/ivadomed/data_spinegeneric_registered/releases/download/r20200907/data_spinegeneric_registered-r20200907.zip
+   unzip ivadomed_spinegeneric_registered.zip
+   # Rename folder
+   mv ivadomed-data_spinegeneric_registered* data_spinegeneric_registered
+
+
+Configuration file
+------------------
+
+In ``ivadomed``, training is orchestrated by a configuration file. Examples of configuration files are available in
+the ``ivadomed/config/`` and the documentation is available in :doc:`../configuration_file`.
```

Comment:
this is actually a problem if people install ivadomed via pip install (which by the way is [what we recommend](https://ivadomed.org/en/cg-tutorial_oneclass/installation.html#install-from-release-recommended)), because in that case they don't have access to the `ivadomed/config`. Should we just alway point to files hosted on GitHub?  People who have cloned the repository will figure out that they also have the file locally.

Reply:
Alternatively, given that these files are copied under the python tree when running pip install (e.g. `venv-ivadomed/lib/python3.6/site-packages/ivadomed/config/config.json`), is there an elegant way to retrieve this path locally? @Drulex or @kousu do you have an idea?

Reply:
Should we change our recommendation for installation, from `pip install` to `git clone`? I feel that for this repo', a git clone should not be a problem for the potential users. What do you think @jcohenadad ?

Reply:
I find it a shame to “downgrade” for a git clone. Why bothering with maintaining our publishing framework then? Also, pypi install is so easy! I’m sure there is a good solution to our problem here